### PR TITLE
POWERMON-199: Use correct kepler namespace in must-gather

### PR DESCRIPTION
**Pull request by vimalk78**
Use labels to get correct namespace name.

### CODE REVIEW

File: must-gather/common [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 set -eu -o pipefail
 
-export KEPLER_NS="kepler-operator"
+export KEPLER_NS=$(oc get ns -l app.kubernetes.io/managed-by=kepler-operator --output=jsonpath='{.items[*].metadata.name}')
```
Comment (sthaha) [inline must-gather/common:20]: perhaps only the first item instead of all items in the list? 
`jsonpath={.items[0].metadata.name}` ?

**Review by sthaha — APPROVED** [review]
I think `-o name` may be

**Comment by sthaha** [comment]
On a related topic, perhaps `status.deploymentNamespace` can be added to kepler.

File: must-gather/common (hunk 1 above)
Comment (vimalk78) [reply must-gather/common:20]: ideally there should be only 1.

Comment (sthaha) [reply must-gather/common:20]: yes, indeed .. already approved.
In a different PR, I think it is better to iterate on all namespaces to capture the details. That way we get all namespaces (perhaps user has both community and power-mon installed) etc ...

**Review by sthaha — APPROVED** [review]

**Pull request merged**